### PR TITLE
setup the slot pattern for UI layout

### LINKED ISSUE #26: Main Layout Implementation
**Is your feature request related to a problem? Please describe.**
This issue will add the layout for the UI's slot pattern.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Created the component files in both JS and SCSS
2. Implemented slot pattern

## Purpose
Implemented the main layout for the UI

Closes #26

### CODE REVIEW

**Review by Gjhnsn — APPROVED** [review]
Looking good, excited to see this!

File: pokeapp/src/Brain.js [hunk 1]
```diff
@@ -106,10 +111,16 @@ export const Brain = () => {
...
-        setAllPokemonFiltered={setAllPokemonFiltered}
+ 
+      <Layout 
+        Navigation={<Navigation/>} 
+        FilterBar={<FilterBar 
+          allPokemon={allPokemon}
+          setAllPokemonFiltered={setAllPokemonFiltered}/>} 
+        SideBar={<SideBar/>}
+        DigitalCardBinder={<DigitalCardBinder/>}
       />
```
Comment (kmausolf) [inline pokeapp/src/Brain.js:122]: YOOOOOOOOO!  This is a GOOD use case for slot pattern.

File: pokeapp/src/Brain.js [hunk 2]
```diff
@@ -106,10 +111,16 @@ export const Brain = () => {
...
-        allPokemon={allPokemon}
-        setAllPokemonFiltered={setAllPokemonFiltered}
+ 
+      <Layout 
+        Navigation={<Navigation/>} 
+        FilterBar={<FilterBar 
+          allPokemon={allPokemon}
+          setAllPokemonFiltered={setAllPokemonFiltered}/>} 
+        SideBar={<SideBar/>}
+        DigitalCardBinder={<DigitalCardBinder/>}
```
Comment (BlackCubes) [inline pokeapp/src/Brain.js:121]: Oh my 😲

File: pokeapp/src/components/Layout.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Layout = ({ Navigation, FilterBar, SideBar, DigitalCardBinder }) => {
+  return (
+    <div className="main-container">
+      {Navigation}
+      {FilterBar}
+      {SideBar}
+      {DigitalCardBinder}
+    </div>
```
Comment (BlackCubes) [inline pokeapp/src/components/Layout.js:10]: Very clean

Comment (kmausolf) [inline pokeapp/src/components/Layout.js:10]: Nice and simple with plenty of room for expansion.  Good choice on using slot pattern (especially if you want to offer your user a selection of layouts!).

File: pokeapp/src/styling/components/_digital-card-binder.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@import '../colorschemes/colors';
+@import '../typography/fonts';
+
+.digital-card-binder {
+    background-color: $component-container;
+    grid-area: 3 / 2 / 4 /3;
+    border: 1px solid black;
+}
```
Comment (Lap343) [inline pokeapp/src/styling/components/_digital-card-binder.scss:8]: Grid I see

**Review by JorgeLVilla — APPROVED** [review]
Looks good! Looking forward to seeing it.

File: pokeapp/src/components/Sidebar.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+const SideBar = () => {
+
+    return(
+        <div className="sidebar">
+            SideBar
+        </div>
+    )
+};
+
+export default SideBar;
```
Comment (BlackCubes) [inline pokeapp/src/components/Sidebar.js:12]: As you add more stuff, this could get bigger, and so you are doing a great job on setting this up

**Review by kmausolf — APPROVED** [review]
Good engineering decisions being made here.  Nice work!

**Review by Lap343 — APPROVED** [review]
nice code

**Review by BlackCubes — APPROVED** [review]
Very good!

**Pull request merged**